.github/workflows/release_build.yml

```diff
@@ -100,21 +100,21 @@ jobs:
     - name: Setup Build (CMake)
       if: matrix.tool == 'cmake'
       run: |
-        mkdir cmake_build
-        cd cmake_build
+        mkdir build
```

Comment:
I thought we did this for dealing with conflicts with Bazel in non-case sensitive OS's?

Reply:
Oh yeah right. That was in another repo that is being built for Windows. I copied this from there but didn't remember why we did this since this repo doesn't build for Windows. I changed this for the unit test run step later but might be able to do it differently.